### ISSUE TITLE
Alternatives within typespec argument lists are always split over multiple lines

### ISSUE THREAD

**Issue by ehamberg** [WhatsApp/erlfmt#89]
erlfmt 0.5.0

(This might be related to #53.)

When formatting with 0.5.0 (after upgrading from 0.4.2), I saw a surprising number of changed files, and it turns out that the changes are typespecs' arguments list where alternatives now always are split over several lines:

```diff
 -spec use_credit(
     store:id(),
     decimal:decimal(),
-    binary() | null,
-    credit | {refund | dispute, binary()},
-    binary() | null
+    binary() |
+    null,
+    credit |
+    {refund | dispute, binary()},
+    binary() |
+    null
 ) -> ok.
```

Is this intentional? I would expect e.g. `binary() | null` to be left on a single line here like in 0.4.2, so this code would remain like this:

```erlang
 -spec use_credit(
     store:id(),
     decimal:decimal(),
     binary() | null,
     credit | {refund | dispute, binary()},
     binary() | null
 ) -> ok.
```

In #53, what happened was that a sequence of alternatives (`|`) needed to be split over multiple lines, and then _all_ alternatives should be on separate lines, but in this case, it's the arguments (`,`) that need to be split into separate lines, not the alternatives _within_ an argument.

Hope this made sense. 😄

**Comment by michalmuskala** [comment]
Thank you for trying erlfmt out! Yes, that definitely looks like a bug

**Comment by awalterschulze** [comment]
And thank you @ehamberg for reporting so quickly :)

**Comment by ehamberg** [comment]
Happy to help! I’m just really happy this project exists! 😄

**Comment by michalmuskala** [comment]
The fix is released in version 0.5.1
